lib/CMakeLists.txt

```diff
@@ -110,10 +110,10 @@ endif()
  
 target_link_libraries(ocpp
     PUBLIC
-        everest::timer
-        websocketpp::websocketpp
+        everest::timer        
         nlohmann_json_schema_validator
         everest::evse_security
+        websocketpp::websocketpp
```

Comment:
That should probably be conditional on LIBOCPP_ENABLE_DEPRECATED_WEBSOCKETPP

Reply:
There's still a find_package call in the toplevel CMakeLists.txt as well: https://github.com/EVerest/libocpp/blob/8a9e06962e4beff69eb954f94b0b28c64bfa8314/CMakeLists.txt#L43
That should be conditional too (and have one for libwebsockets instead)

Reply:
I have done just that, however we have the following problem: Since at the moment the interface refactor is NOT done (in the current state) ALL the code still depends on websocket++ headers. So, I've tried to make it conditional but the compilation failed when the websocket++ library was not added.

The workaround is to leave it inside, until we have the interface refactor that will not make the code dependant on websocket++.